### ISSUE TITLE
How to get /update  specified property of one section

### ISSUE THREAD

**Issue by masterchen** [JeevanJames/IniFile#9]
I assume  i have ini content as follow:
//////////
[tcp]
ip = localhost
/////////

then read the file:

 string iniStr = File.ReadAllText(path);
            Ini ini = Ini.Load(iniStr)

then how to get property value of "ip" , thanks

By the way ,  when i use "Ini ini = new ini(file)" to load ini file , there is always error : can't reference null object.  im sure the file is at the right folder. can you give a hint ?

**Comment by JeevanJames** [comment]
@masterchen Thanks for reporting this bug. There is indeed a null reference exception being thrown from the `Ini` constructor. This has been fixed and a patch release 1.5.1 made on [NuGet](https://www.nuget.org/packages/IniFile.NET/1.5.1).

To answer your other question, you can use indexer syntax to access the property:
```cs
string ip = ini["tcp"]["ip"];
```

If you're not sure whether the `tcp` section exists, you can use the `TryGetValue` method to check first:
```cs
string ip = "default value";
if (ini.TryGetValue("tcp", out var tcp))
    ip = tcp["ip"];
```

In the next point release (1.6), I'm planning to add support for the C# 6 null conditional operator:
```cs
// Will not work now
string ip = ini["tcp"]?["ip"];
```

**Comment by masterchen** [comment]
Thanks i will try

**Comment by JeevanJames** [comment]
@masterchen I've just published version [1.6.0](https://www.nuget.org/packages/IniFile.NET/1.6.0), which supports C# 6 null conditional operators:

```cs
string ip = ini["tcp"]?["ip"];
if (ip == null)
    // tcp section does not exist
```

Pre-C# 6 usage:
```cs
var tcp = ini["tcp"];
if (tcp != null)
    string ip = tcp["ip"];
```

I'll be closing this issue for now. Please re-open or create a new issue if you face any further problems.
